walrus/walrus-bmi-psfetch.ipynb

```diff
@@ -57,17 +62,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 21,
    "metadata": {},
    "outputs": [],
    "source": [
-    "model = BmiClientDocker(image=models['WALRUS']['docker'], image_port=55555, \n",
-    "                        input_dir=\"./input\")"
+    "model = BmiClientDocker(image='ewatercycle/walrus-grpc4bmi:v0.2.0', image_port=55555, \n",
+    "                        input_dirs=(\"./input\",))"
```

Comment:
Please add `work_dir` to BmiClientDocker().

Reply:
done